example/lib/common/ui/organisms/degrade_tile.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:flutter/material.dart';
+import 'package:hmssdk_flutter_example/common/util/utility_function.dart';
+import 'package:hmssdk_flutter_example/meeting/peer_track_node.dart';
+import 'package:provider/provider.dart';
+
+class DegradeTile extends StatefulWidget {
+    final double itemHeight;
+  
+  final double itemWidth;
+  DegradeTile(
+      {Key? key,this.itemHeight = 200,
+      this.itemWidth = 200,})
+      : super(key: key);
+
+  @override
+  State<DegradeTile> createState() => _DegradeTileState();
+}
+
+class _DegradeTileState extends State<DegradeTile> {
+  @override
+  Widget build(BuildContext context) {
+    return Selector<PeerTrackNode, bool>(
+        builder: (_, data, __) {
+          
+            return Visibility(
+              visible: data,
+                child: Container(
+                  height: widget.itemHeight + 110,
+                      width: widget.itemWidth - 4,
+                      decoration: BoxDecoration(
+                        color: Colors.grey[800],
+                          borderRadius: BorderRadius.all(Radius.circular(10))),
+                  child: Center(
+                      child: CircleAvatar(
+                          backgroundColor: Utilities.colors[context
+                                  .read<PeerTrackNode>()
+                                  .peer
+                                  .name
+                                  .toLowerCase()
+                                  .codeUnitAt(0) %
```

Comment:
we can make this more simple by performing these operations in the Utilities file itself by just passing the name from here.